stetho/src/main/java/com/facebook/stetho/common/android/ViewUtil.java

```diff
@@ -18,47 +20,85 @@
   private ViewUtil() {
   }
 
+  private static boolean isTouchable(View view) {
+    if (view.getVisibility() != View.VISIBLE) {
+      return false;
+    }
+
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
```

Comment:
nit: this is the anti-pattern I discussed earlier that has the tendency to create perf problems when you aren't very careful about protecting the Dalvik verifier from seeing illegal code.  The pattern which makes this simple to avoid is the same as one you've used elsewhere:
```java
abstract class ViewGetAlphaCompat {
  @Nullable private static final ViewGetAlphaCompat sInstance;

  static {
    if (honeycomb+) {
      sInstance = new HoneycombAndBeyondImpl();
    } else {
      // Can't be done...
      sInstance = null;
    }
  }

  @Nullable
  public static ViewGetAlphaCompat getInstance() {
    return sInstance;
  }

  public float getAlpha(View v);

  @TargetApi(HONEYCOMB)
  private static class HoneycombAndBeyondImpl { ... }
}
```